### ISSUE TITLE
API response time in Rapidoc

### ISSUE THREAD

**Issue by kle97** [mrin9/RapiDoc#533]
In Swagger UI, I can have a response time display for each endpoint by enabling ```displayRequestDuration```. Do we have similar feature in Rapidoc?
In case we don't have one, I don't mind writing additional script with ```before-try``` and ```after-try```, but I don't have any clear idea to show the response time. Do you have any suggestion?

**Comment by mrin9** [comment]
> Do we have similar feature in Rapidoc?

No we dont

> Do you have any suggestion

you can put a `<div>`  element  in  `<slot>` lets say on the header.
and then update this elements body with result from your calculation.

Alternatively  if you can provide a PR for this functionality I will accept it , that way it will be much cleaner solution

**Comment by kle97** [comment]
@mrin9 I wrote additional script with ```before-try``` and ```after-try``` to measure the response time. Yes, I used <slot> element to show the respone time on the header. 
However, I realize the response times I got are not very accurate (error from a few to a hundred milliseconds), compared to the response times in Chrome devtools's network tab. And the response time returns by ```displayRequestDuration``` in Swagger UI is even more inaccurate than mine. 
Therefore, I don't think it's a good idea to make response time as a feature for rapidoc, although it gives some sense for developer about the duration of each requests.
